shared-package.props

```diff
@@ -24,9 +24,8 @@
 
   <PropertyGroup>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
-    <IncludeSymbols>true</IncludeSymbols>
-    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
-    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+    <DebugType>embedded</DebugType>
```

Comment:
I like this. Instead of adding .pdb files in the package (the default), the symbols now end up directly in the DLL. I've experienced hard-todiagnose issues in the past with coverage tools not properly resolving the absolute/relative paths (either only working locally, or only working in cibuild), and this solves the whole problem. Another nice side effect is that it avoids the annoying confirmation prompt when stepping into.

With this setting in place, `ContinuousIntegrationBuild` can be removed.

Reply:
Why remove `ContinuousIntegrationBuild`? I would think the pain around absolute/relative path differences was already resolved in all the tooling... I don't see what else it affects, but [the docs](https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#continuousintegrationbuild) imply that isn't the only thing that changes

Reply:
Well, the reason for keeping it seems more of a theoretical concern (see https://github.com/dotnet/roslyn/issues/55860#issuecomment-905018564), but I'm fine leaving it in.